manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -217,6 +217,9 @@ global:
     #   cpu: 100m
     #   memory: 128Mi
 
+  # enable validating webhooks to ensure that CRDs created are valid
+  configValidation: true
```

Comment:
can we prohibit disabling validation at all

Reply:
We'd need to do this for istioctl based installs as well. Maybe we remove this completely? I don't know how this is used